ruffd-core/src/server_ops.rs

```diff
@@ -13,8 +13,6 @@ use std::sync::Arc;
 // TODO macro the unwrapping of state_handles
 
 fn message_into_diagnostic(msg: Message) -> lsp_types::Diagnostic {
-    // As ruff currently doesn't support the span of the error,
-    // only have it span a single character
```

Comment:
I believe these comments are stale.

Reply:
Definitely stale, there's more stale comments around I'll look to clean up, most notably around TODOs